### PR TITLE
deploy: change operator image to internal repo

### PR DESCRIPTION
**Pull request by jhrozek**
This makes a in-cluster deployment easier with make image-to-cluster

### CODE REVIEW

**Comment by jhrozek** [comment]
Same as with the other PR:
```
level=error msg="Error: Error creating VPC Endpoint: VpcEndpointLimitExceeded: The maximum number of VPC endpoints has been reached."
level=error msg="\tstatus code: 400, request id: 98026b6e-c763-4ef2-8e0b-2fb65304a7f0"
level=error
level=error msg="  on ../tmp/openshift-install-135359444/vpc/vpc.tf line 21, in resource \"aws_vpc_endpoint\" \"s3\":"
level=error msg="  21: resource \"aws_vpc_endpoint\" \"s3\" {"
level=error
level=error
level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply using Terraform"

```

File: deploy/operator.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: compliance-operator
           # Replace this with the built image name
-          image: quay.io/jhrozek/compliance-operator
```
Comment (JAORMX) [inline deploy/operator.yaml]: Lets not change this just yet.

**Comment by JAORMX** [comment]
CI is already using the images built in CI. I'm unsure why this is needed. Lets discuss next week.

**Pull request closed without merging**